src/patterns-hmrc-govuk-team/factsheets/index.njk

```diff
@@ -0,0 +1,116 @@
+
+<p class="govuk-body">
+Use this pattern when creating or amending factsheets.
+</p>
+
+<h2 class="govuk-heading-l" id="how">How to design tax avoidance and compliance factsheets using this pattern</h2>
+
+<h3 class="govuk-heading-m" id="type">Document type</h3>
+
+<p class="govuk-body">Publication: guidance (with an HTML attachment)</p>
+
+<h3 class="govuk-heading-m">Publication</h3>
+
+<h4 class="govuk-heading-s" id="title">Title</h4>
+
+<p class="govuk-body">The title should be ‘Compliance checks: xxx [description] — CC/FSxx [factsheet number]’.</p>
+
+<p class="govuk-body">For example, ‘Compliance checks: electronic sales suppression — CC/FS68’.</p>
+
+<h4 class="govuk-heading-s" id="summary">Summary</h4>
+
+<p class="govuk-body">Begin with ‘This factsheet xxx.’. Then read the document and summarise the factsheet’s content.</p>
+
+<p class="govuk-body">For example, ‘This factsheet tells you about the penalties HMRC may charge if you are involved in electronic sales suppression (ESS).’.</p>
+
+<h4 class="govuk-heading-s">Body</h4>
+
+<p class="govuk-body">Read the document and summarise the factsheet’s content to create the body text.</p>
+
+<p class="govuk-body">It will also normally include, ‘Factsheets are for guidance only and reflect HMRC’s position at the time of writing.’.</p>
+
+<h3 class="govuk-heading-m">HTML attachment</h3>
+
+<h4 class="govuk-heading-s">Title</h4>
+
+<p class="govuk-body">Use the description that you’ve used in the title of the stub page. The titles can be similar but cannot be exactly the same.</p>
+
+<p class="govuk-body">[Description] — CC/FSxx [factsheet number]</p>
+
+<p class="govuk-body">For example, ‘Electronic sales suppression — CC/FS68’.</p>
+
+<h4 class="govuk-heading-s">Body</h4>
+
+<p class="govuk-body">Enter the contents of the factsheet.</p>
+
+<p class="govuk-body">Do not include numbers in section headings.</p>
+
+<p class="govuk-body">For a live example check <a href="https://www.gov.uk/government/publications/compliance-checks-electronic-sales-suppression-ccfs68" class="govuk-link">compliance checks: electronic sales suppression — CC/FS68</a>.</p>
+
+<h3 class="govuk-heading-m">Collection page</h3>
+
+<p class="govuk-body">Add the URL of any new page to the <a href="https://www.gov.uk/government/collections/hm-revenue-and-customs-compliance-checks-factsheets" class="govuk-link">HMRC compliance checks factsheets document collection</a>. Do not create a public change note.
+</p>
+
+<h2 class="govuk-heading-l" id="how">How to design other factsheets using this pattern</h2>
+
+<h3 class="govuk-heading-m" id="type">Document type</h3>
+
+<p class="govuk-body">Publication: guidance (with an ODT, or PDF plus ODT attachment)</p>
+
+<h3 class="govuk-heading-m">Publication</h3>
+
+<h4 class="govuk-heading-s" id="title">Title</h4>
+
+<p class="govuk-body">[Description]</p>
+
+<p class="govuk-body">For example, 'Tax credits: leaving the UK'.</p>
+
+<h4 class="govuk-heading-s" id="summary">Summary</h4>
```

Comment:
This ID already exists. Maybe change both to be more specific

Reply:
Good spot @timsb. I've just prefixed the 'other-' ones, as otherwise would need something long like 'avoidance-and-compliance' adding and I don't think it adds anything in practice.